src/__tests__/__snapshots__/Header.test.js.snap

```diff
@@ -0,0 +1,18 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
what is this file for?

Reply:
https://jestjs.io/docs/en/snapshot-testing